x-pack/plugins/cases/server/client/metrics/actions/aggregations/isolate_host.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { IsolateHostActionType } from '../../../../../common/api';
+import { CASE_COMMENT_SAVED_OBJECT } from '../../../../../common/constants';
+import { AggregationBuilder, AggregationResponse } from '../../types';
+
+interface ActionsAggregation {
+  actions?: {
+    buckets: Array<{ key: string; doc_count: number }>;
+  };
+}
+
+export class IsolateHostActions implements AggregationBuilder {
+  constructor(private readonly uniqueValuesLimit: number = 10) {}
```

Comment:
Could you add a comment that `uniqueValuesLimit` will need to be greater than or equal to the number of possible values for `actions.type` (currently only 2 I believe), otherwise we'll miss some information.

Reply:
I was not sure if we should parametrize the limit for this aggregation, the `action.type` values are static and controlled, it makes more sense to do it when we have dynamic values such as the alert hosts, users... Anyway, I added the comment.

Reply:
Yeah good point, we could move it to a static variable if you wanted.

Reply:
Yeah, good point. I agree.